src/drivers/PortFolder/KMDFTestTemplate/Driver.c

```diff
@@ -0,0 +1,171 @@
+/*++
+
+Module Name:
+
+    driver.c
+
+Abstract:
+
+    This file contains the driver entry points and callbacks.

```

Comment:
You should have a note somewhere in these comments that this template file is designed to be used with code snippets for CodeQL analysis.

Reply:
comment added.